.gemini/config.yaml

```
@@ -0,0 +1,9 @@
+have_fun: false
+code_review:
+  disable: false
+  comment_severity_threshold: MEDIUM
+  max_review_comments: -1
+  pull_request_opened:
+    help: false
```

```suggestion
    help: true
```

after discussion with Julie we decided to keep this in for now.